sass/layouts/services.scss

```diff
@@ -0,0 +1,45 @@
+.services {
+  background-color: $color-services-background;
+  border-bottom: 6px solid $color-services-shadow;
+  position: relative;
+  top: 66px;
+  @include media-768 {
+    top: 47px;  
+    border-bottom: none;
+  }
+}
+
+.services-header {
+  padding: 59px 44px 60px 40px;
+  box-sizing: padding-box;
+  @include media-1020 {
+    padding: 39px 41px 28px 42px;
+  }
+  @include media-768 {
+    padding: 25px 30px;
+  }
+  @include media-375 {
+    padding: 25px 15px;
+  }
+}
+
+.services-header p {
+  margin-top: 18px;
+}
+
+.services-list {
+  padding: 0 40px 60px;
+  @include media-768 {
+    padding: 0 33px 15px;
+  }
+  @include media-375 {
+    padding: 0 15px 33px;  }
+}
+
+.tool p {
+  @include font(1.2rem, normal, 2.2rem);
+  text-align: justify;
+}
+.tool h3 {
+  margin: 0 0 7px 0;
```

Comment:
You can rewrite this using only 3 values.

Reply:
DONE!